shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -48,6 +48,10 @@
 // The parent view handles clipping to its subviews.
 @interface ChildClippingView : UIView
 
+// Adds a blur filter to its layers. Returns NO if Apple's API has changed and
+// blurred backdrop filters cannot be applied, otherwise returns YES.
+- (BOOL)applyBlurBackdropFilters:(NSArray*)blurRadii;
```

Comment:
Can we test this method returns NO(and not crash) if the implementation of `UIVisualEffectView` changes? 
We might need to make this method back to an instance method and injecting a `UIVisualEffectView` that doesn't contain a `_UIVisualEffectBackdropView`, or doesn't contain the "gaussianBlur" filter.